developers/weaviate/connections/connect-query.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: Connect with Query App
+sidebar_position: 60
+image: og/docs/connect.jpg
+# tags: ['getting started', 'connect']
+---
+
+import WCDQueryConsoleLocation from '/developers/wcs/img/wcs-query-console-location.jpg';
+
+The Weaviate Cloud (WCD) [query application (query app)](/developers/wcs/console) is a browser-based GraphQL IDE. Use the query app to work interactively with your WCD clusters.
+
+## Open the query app
+
+To use the query app, open your [WCD Dashboard](https://console.weaviate.cloud).
+
+In the left-hand menu, click the Query app icon.
+
+<img src={WCDQueryConsoleLocation} width="25%" alt="WCD query app location"/>
+
+## Select a Weaviate instance
+
+The query app connects directly to your WCD clusters. The query app can also connect to other Weaviate instances if you have anonymous access enabled.
+
+To select a cluster, Click the `Query` button and choose a cluster from the clusters list.
+
```

Comment:
Can we add a query example to try?
It could be a fetch (Get without a nearX operator) example, so the keys won't be needed.

Reply:
Updated